timescaledb/how-to-guides/distributed-hypertables/index.md

```diff
@@ -34,7 +34,7 @@ fetching the raw data from data nodes and performing the join locally.
 Distributed hypertables currently have some limitations when
 compared to non-distributed hypertables. Before creating a
 distributed hypertable for production workloads, please review our
-[limitations](/latest/overview/limitations/#distributed-hypertable-limitations)
+[limitations](https://docs.timescale.com/timescaledb/latest/overview/limitations/)
```

Comment:
I've tried fixing this a number of different ways over several attempts. I'm pretty sure this is how it was last time, but I'm willing to try anything at this point.

Reply:
There were two major issues with this, one totally my bad and one that has to do with how the link is written.  There used to be an error that would occur when a hash was included in the url - that has been resolved on my end. The part that's broken here is that it has to start with the product - ESPECIALLY if the version is given. If neither the product or version is given, the algo will now add the product and version the user is on. If it has both product and version, it gets passed on no problem. If it is missing one but not the other, it will have an extra version or product added onto it. In terms of the full link here, I don't believe it's actually necessary. It won't break anything, but it will miss out on the caching gatsby does for internal links, as it will be treated as an external link. Hopefully this clears things up!

Reply:
Moving forward, it's always best practice to include product and version -- I've built in flexibility to allow for some variation, but I think including product and version should be the go to.

Reply:
😵

Reply:
```suggestion
[limitations](/timescaledb/latest/overview/limitations/)
```